SemanticUI/Modules/Dropdown/Select.elm

```diff
@@ -0,0 +1,327 @@
+module SemanticUI.Modules.Dropdown.Select exposing
+    ( Builder
+    , Config
+    , Select(..)
+    , Variation(..)
+    , attributes
+    , button
+    , caret
+    , disabled
+    , fluid
+    , inline
+    , linkItem
+    , scrolling
+    , select
+    , toCustomHtml
+    , toHtml
+    , toItem
+    , toggleEvent
+    )
+
+{-| A [dropdown](https://semantic-ui.com/modules/dropdown.html) with selectable options in its drawer.
+
+Example of `Select.select` :
+
+    Select.select
+        { identifier = "nav-select"
+        , onToggle = ToggleNavSelect
+        , onSelect = identity
+        , drawerState = model.navDrawerState
+        }
+        |> Select.toHtml
+            { options = [ NavHome, NavBack ]
+            , optionLabel = text << navToString
+            }
+
+See `SematicUI.Modules.Dropdown.Selection` for dropdowns that indicate a current selection state.
+
+-}
+
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import Html.Builder as Html
+import Html.Events exposing (..)
+import SemanticUI.Elements.Button as Button
+import SemanticUI.Modules.Dropdown as Dropdown exposing (Dropdown(..))
+import SemanticUI.Modules.Dropdown.Drawer as Drawer
+import SemanticUI.Modules.Dropdown.Toggle as Toggle
+
+
+{-| Select variations. For internal use only.
+
+Similar to `Dropdown.Variation`
+
+-}
+type Variation msg
+    = Ordinary
+    | Button (Button.Config msg)
+    | Inline
+    | Selection { compact : Bool }
+
+
+variationClassList : Variation msg -> List ( String, Bool )
+variationClassList variation =
```

Comment:
How about `variationAttributes : Variation msg -> List (Attribute msg)`? I suggest this because it tidies up the `(  "selection", True )` element, into just `class "selection"`.

Reply:
Variations actually have attributes and classes which I started dealing with separately starting from this commit: https://github.com/circuithub/elm-semantic-ui/commit/d9a54d54c35ce581ee2d641aee4b5771cc76eea0.

The main problem is that you have to be rather careful with the order of classes and if memory serves me it's actually sort of the opposite of what you want for attributes.

In theory I could make this `List [ String ]`, but `classList` feels to me a bit more idiomatic to elm.